src/js/components/Select/SelectContainer.js

```diff
@@ -36,22 +40,60 @@ const SelectOption = styled(Button)`
   width: 100%;
 `;
 
-const ClearButton = ({ clear, onClear, name, theme, setFocus }) => {
+const ClearButtonContainer = styled(Box)``;
+const ClearButtonText = styled(Text)``;
+const ClearButtonComponent = ({
+  className,
+  clear,
+  onClear,
+  name,
+  theme,
+  setFocus,
+}) => {
   const { label, position } = clear;
   const align = position !== 'bottom' ? 'start' : 'center';
   const buttonLabel = label || `Clear ${name || 'selection'}`;
+
   return (
     <Button
+      // className must be passed for styled-components component interpolation
+      // https://styled-components.com/docs/advanced#referring-to-other-components
+      className={className}
       onClick={onClear}
       onFocus={() => setFocus(true)}
       onBlur={() => setFocus(false)}
     >
-      <Box {...theme.select.clear.container} align={align}>
-        <Text {...theme.select.clear.text}>{buttonLabel}</Text>
-      </Box>
+      <ClearButtonContainer {...theme.select.clear.container} align={align}>
+        <ClearButtonText {...theme.select.clear.text}>
```

Comment:
Same as above comment, could we just spread the theming directly on the Text?
```suggestion
        <Text {...theme.select.clear.text}>
```